### ISSUE TITLE
Double.parseDouble(...) != FastDoubleParser.parseDouble(...)

### ISSUE THREAD

**Issue by fmeum** [wrandelshofer/FastDoubleParser#7]
I have found another input string for which the return values of `Double.parseDouble` and `FastDoubleParser.parseDouble` differ. This one is less important than #6 though as it implies only a very minor loss in precision:

```
Double.parseDouble("-2.2222222222223e-322"): -2.2E-322
FastDoubleParser.parseDouble("-2.2222222222223e-322"): 0.0
```

Both this issue and #6 have been found with the open-source JVM fuzzer [Jazzer](https://github.com/CodeIntelligenceTesting/jazzer). If you are interested in these kinds of findings, I could add the fuzzer to the project as a PR.

**Comment by wrandelshofer** [comment]
Thank you for finding this bug!

I do not want to include any more third party dependencies to the library. Can Jazzer be used without integrating into the library? 
(I just noticed, I still haven't properly configured the dependency to JUnit 5 in the project yet - doh).

**Comment by fmeum** [comment]
You can run Jazzer yourself by executing `./run_jazzer.sh` on the [jazzer branch of my fork](https://github.com/CodeIntelligenceTesting/FastDoubleParser/tree/jazzer). Note that this requires Linux, so you might need to run this within Docker if you are developing on another system. You can make changes to the fuzzer in https://github.com/CodeIntelligenceTesting/FastDoubleParser/blob/jazzer/src/ch/randelshofer/Fuzzer.java, it's just Java.

**Comment by lemire** [comment]
Please see https://github.com/wrandelshofer/FastDoubleParser/issues/9 where I report on many such problems.

**Comment by wrandelshofer** [comment]
I have fixed this bug in dfdd66e26fb42612e7e85ca734208a4dfbae8d69.

I am going to try out jazzer !
